core/templates/components/state-editor/state-responses-editor/state-responses.directive.ts

```diff
@@ -578,7 +578,7 @@ angular.module('oppia').directive('stateResponses', [
             $scope.stateSolicitAnswerDetailsService = (
               StateSolicitAnswerDetailsService);
             $scope.dragDotsImgUrl = UrlInterpolationService.getStaticImageUrl(
-              '/general/drag_dots.png');
+              '/general/drag_dots.webp');
```

Comment:
Could we support both PNG and WebP here?

Reply:
@darksun27  ping!